com.microsoft.mrtk.input/Tests/Runtime/Utilities/InputTestUtilities.cs

```diff
@@ -115,6 +115,23 @@ public static void RotateCamera(Vector3 rotationDelta)
             hmd.Update(Vector3.zero, rotationDelta);
         }
 
+        /// <summary>
+        /// Returns a position placed in front of the user's head, offset forward by the given distance.
+        /// </summary>
+        public static Vector3 InFrontOfUser(float distanceFromHead = 0.4f)
+        {
+            return Camera.main.transform.position + Camera.main.transform.forward * distanceFromHead;
+        }
+
+        /// <summary>
+        /// Returns a position placed in front of the user's head, offset by the given vector, which
+        /// is transformed by the XROrigin's camera offset object's reference frame.
+        /// </summary>
+        public static Vector3 InFrontOfUser(Vector3 offset)
+        {
+            return Camera.main.transform.position + PlayspaceUtilities.XROrigin.CameraFloorOffsetObject.transform.TransformVector(offset);
```

Comment:
This is so that the passed-in offset is computed as relative to the playspace's coordinate system... am I having a brainfart? Is this not what I'm supposed to do?

It's the same thing effectively as `Camera.main.transform.position + Camera.main.transform.TransformVector(offset)`, except that would make the offset reference frame influenced by the head pose, instead of the reference frame of the playspace.

Reply:
Changed to now be totally camera-relative